f5/bigip/tm/security/ip_intelligence.py

```diff
@@ -110,10 +106,6 @@ def __init__(self, blacklist_categorys):
         super(Blacklist_Category, self).__init__(blacklist_categorys)
         self._meta_data['required_json_kind'] = \
             'tm:security:ip-intelligence:blacklist-category:blacklist-categorystate'
-        self._meta_data['required_creation_parameters'].update(
```

Comment:
is the partition parameter mandatory for this if so only name should have been removed and partition should be added